.github/workflows/test.yaml

```diff
@@ -0,0 +1,9 @@
+name: Tests
+
+on:
+  pull_request:
+
+jobs:
+  unit-tests:
+    uses: canonical/operator-workflows/.github/workflows/test.yaml@cea2ac306b4f4c1475d73b1a4c766d62e5b1c8a9
```

Comment:
Try running the latest commit of `test.yaml`. We fixed the commit at the time due to instabilities with the workflow at the time.